WooCommerce/WooCommerceTests/Extensions/IconsTests.swift

```diff
@@ -364,4 +364,8 @@ final class IconsTests: XCTestCase {
     func test_wordPressLogoImage_is_not_nil() {
         XCTAssertNotNil(UIImage.wordPressLogoImage)
     }
+
+    func testErrorImageIsNotNil() {
```

Comment:
nit: we use snake case for test cases now based on the [doc](https://github.com/woocommerce/woocommerce-ios/blob/develop/docs/naming-conventions.md#test-methods) 😄 
```suggestion
    func test_errorImage_is_not_nil() {
```

Reply:
Thank you! Resolved in https://github.com/woocommerce/woocommerce-ios/pull/3307/commits/e0ff816f2e45f294cc7b6ebd063078852c826f24